components/forms/Login/index.jsx

```diff
@@ -1,15 +1,18 @@
 import Link from 'next/link';
 
+import Button from '@/components/Button';
+import Input from '@/components/Input';
+
 export const loginFormTestIds = {
-  form: 'login-form',
-  emailLabel: 'email',
-  emailField: 'input-email',
-  passwordLabel: 'password',
-  passwordField: 'input-password',
-  loginButton: 'btn-login',
+  form: 'login__form',
+  emailLabel: 'login__email',
+  emailField: 'login__input-email',
+  passwordLabel: 'login__password',
+  passwordField: 'login__input-password',
+  loginButton: 'login__button',
```

Comment:
How about renaming it like this so it component name?

```suggestion
  form: 'login-form',
  emailLabel: 'login-form__email',
  emailField: 'login-form__input-email',
  passwordLabel: 'login-form__password',
  passwordField: 'login-form__input-password',
  loginButton: 'login-form__button',
```

Reply:
done in d129258d877c1e23ba31313cc0390ad9627eb192